src/olcs/FeatureConverter.js

```diff
@@ -340,10 +340,14 @@ exports.prototype.olCircleGeometryToCesium = function(layer, feature, olGeometry
  * @param {!Array<Cesium.Cartesian3>} positions The vertices of the line.
  * @return {!Cesium.GroundPrimitive} primitive
  */
-exports.prototype.createStackedGroundCorridors = function(layer, feature, width, color, positions) {
-  let previousDistance = 0;
+exports.prototype.createStackedGroundCorridors = function(width, color, positions) {
```

Comment:
You should keep the `layer and `feature` parameters.
They allow projects overriding the `createStackedGroundCorridors` method to have different behaviour depending on the actual layer/feature being converted.

Reply:
That makes sense.